impl/src/planning.rs

```diff
@@ -645,13 +648,193 @@ mod tests {
       )
       .unwrap();
 
+    let wasm_version = Version::parse("0.2.68").unwrap();
+
     // Vendored builds do not use binary dependencies and should not alter the outputs
     assert!(planned_build
       .crate_contexts
       .iter()
-      .find(|ctx| ctx.pkg_name == "wasm-bindgen-cli" && ctx.pkg_version == "0.2.68")
+      .find(|ctx| ctx.pkg_name == "wasm-bindgen-cli" && ctx.pkg_version == wasm_version)
       .is_none());
   }
+
+  #[test]
+  fn test_semver_matching() {
```

Comment:
can this be broken up into a couple, more specific, tests?

Reply:
I would like to just commit as I am getting exhausted with this PR

Reply:
fair enough